core/livepeernode.go

```diff
@@ -119,8 +122,17 @@ func (n *LivepeerNode) CreateTranscodeJob(strmID StreamID, profiles []ffmpeg.Vid
 		return err
 	}
 
-	blk, err := b.BlockByNumber(context.Background(), nil)
-	if err != nil {
+	var blk *types.Block
+	getBlock := func() error {
+		blk, err = b.BlockByNumber(context.Background(), nil)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+	bo := backoff.NewExponentialBackOff()
+	bo.MaxElapsedTime = time.Second * 15
+	if err := backoff.Retry(getBlock, backoff.WithMaxRetries(bo, SubscribeRetry)); err != nil {
```

Comment:
In the case of websocket connections to Infura, we really should retry indefinitely. We don't want to force user interaction if all they need to do is re-establish the connection.

Reply:
Fair enough.  My concern was around spamming the Infura network, but I guess they are there for a reason and we should always try to reconnect.